models/response.js

```diff
@@ -0,0 +1,56 @@
+var User = require('./user');
+var Service = require('./request');
+module.exports = function(sequelize, DataTypes) {
+    var Response = sequelize.define("Response", {
+        responder_id :{
+            type: DataTypes.INTEGER,
+            primaryKey:true,
+            allowNull: false,
+            autoIncrement: true,
+            unique: true            
+        },
+        request_id :{
+            type: DataTypes.INTEGER,
+            primaryKey:true,
+            allowNull: false,
+            autoIncrement: true,
+            unique: true            
+        },
```

Comment:
These won't be necessary anymore since the Sequelizer associations generate the foreign keys for us